src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -648,26 +651,27 @@ public static boolean verifyBag(final File bagDir) {
     /**
      * Method to find and set the repository root from the resource uri.
      * @param uri the URI for the resource
-     * @throws IOException
-     * @throws FcrepoOperationFailedException
      */
-    private void findRepositoryRoot(final URI uri) {
-        repositoryRoot = uri;
+    @VisibleForTesting
+    public URI findRepositoryRoot(final URI uri) {
```

Comment:
The following test does not seem like an integration test (i.e. it is not actually making any HTTP requests):
https://github.com/fcrepo4-labs/fcrepo-import-export/blob/0fab8b4e9b850605e557d42ac3007563760396d9/src/test/java/org/fcrepo/importexport/integration/ImporterIT.java#L283

But if you want to keep it in `ImporterIT.java`, I can respect that.
Let me know either way, and we can either move forward with the code as it stands now, or with another commit that moves the mentioned test method to `ImporterTest.java` and reduces access on this method to `protected`.